flink-table/flink-table-runtime/src/test/java/org/apache/flink/table/formats/raw/RawFormatSerDeSchemaTest.java

```diff
@@ -197,12 +247,12 @@ public static TestSpec type(DataType fieldType) {
             return new TestSpec(fieldType);
         }
 
-        public TestSpec value(Object value) {
-            this.value = value;
+        public TestSpec values(Object[] values) {
```

Comment:
make this a var arg to avoid the need for `new X[]{}` in the test specs. this will improve code readibility.